tests/test_special_euclidean.py

```diff
@@ -159,7 +159,6 @@ def test_random_and_belongs(self):
         base_point = self.group.random_uniform()
         result = self.group.belongs(base_point)
         expected = gs.array([True])
-        print(result, expected)
         self.assertAllClose(result, expected)
 
     # @geomstats.tests.np_and_tf_only
```

Comment:
Left-over comment (I know it's not your code, but could you still delete it?)

Reply:
It might be mine, as I was looking at which tests now worked with Pytorch and Tensorflow. I just forgot to remove them when doing the PR.
Same as above, in the next one they're gone.

Reply:
OK! Can't wait for this next one :p